### DISCUSSION TITLE
Using OpenHeart

### DISCUSSION THREAD

**Discussion by muan** [dddddddddzzzz/OpenHeart#4] [Show and tell]
Hi all! I'd like to start a thread where we just share the OpenHeart implementations (and clients) that we have, so that we get 1. more feeds to follow, 2. we see how everyone's using it.

I'll start.

Where to see it: https://muan.co/
Built with [`<open-heart>`](https://github.com/dddddddddzzzz/open-heart-element) and a [custom stylesheet](https://github.com/muan/site/blob/e471298f5caf4cc13d5b501cbee5dfae02339f88/assets/new.scss#L687-L708).

**Comment by javierarce** [comment]
Hi! I implemented OpenHeart in the [photo gallery](https://javier.computer/photos) of my blog.

Right now, my [implementation](https://javier.computer/scripts/hearts.js?version=5.0.52) has the counter enabled by default, but I like the way @muan triggers the `getCount` method through the `IntersectionObserver` so I'll probably apply the same solution.

**Comment by muan** [comment]
OMG Spotted in the wild 😍 

https://jlelse.blog by @jlelse
https://www.benji.dog/articles/interactions-or-reactions/ by @benjifs
https://tracydurnell.com/2024/04/26/adding-a-heart-reacji-to-my-wordpress-theme/
https://axiixc.com/blog/2024-05-08-trying-out-open-heart/ @axiixc
https://jonas.brusman.se by @himynameisjonas

**Reply by javierarce** [reply]
Found another one: https://alexsirac.com/openheart-protocol-installed/

**Reply by jlelse** [reply]
I'm not exactly using OpenHeart in GoBlog, but the concept is quite similar.

**Comment by abhin4v** [comment]
I added Openhearts in my blog posts, for example: https://abhinavsarkar.net/posts/nix-for-haskell/.
The backend is Cloudflare workers and KV namespaces and the frontend is entirely custom HTML/CSS/Javascript, all written by me.
